### PR TITLE
Java sample code

### PR DESCRIPTION
**Pull request by dpgraham**
* Written in IntelliJ and uses TestNG framework (which is basically an advanced version of JUnit)

### CODE REVIEW

File: sample-code/java/src/AndroidBasicInteractionsTest.java [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+        capabilities.setCapability("app", app.getAbsolutePath());
+        capabilities.setCapability("appPackage", PACKAGE);
+        capabilities.setCapability("appActivity", SEARCH_ACTIVITY);
+        driver = new AndroidDriver<WebElement>(service.getUrl(), capabilities);
+    }
+
+    @AfterSuite
+    public void tearDown() {
+        driver.quit();
+    }
```
Comment (SrinivasanTarget) [inline sample-code/java/src/AndroidBasicInteractionsTest.java]: Ideally we should stop the service too. @dpgraham Have another query, are we running these tests in Travis? Or it's just available as sample for reference?

**Review by SrinivasanTarget — CHANGES_REQUESTED** [review]

**Comment by saikrishna321** [comment]
@dpgraham @jlipps as we are getting new sample code in place for all clients, can we start using the RN app as test application (https://github.com/appium/VodQAReactNative)

https://github.com/appium/appium/issues/6098

File: sample-code/java/src/AndroidBasicInteractionsTest.java (hunk 1 above)
Comment (dpgraham) [reply sample-code/java/src/AndroidBasicInteractionsTest.java]: This is just for sample but if we could run it in Travis that would be a bonus.

Comment (SrinivasanTarget) [reply sample-code/java/src/AndroidBasicInteractionsTest.java]: @dpgraham we can do that too.

**Comment by dpgraham** [comment]
Fixes pushed.

**Review by SrinivasanTarget — APPROVED** [review]

**Pull request merged**